Makefile

```diff
@@ -1,6 +1,7 @@
 # Common User-Settable Flags
 # ==========================
-PREFIX?=gcr.io/google_containers
+# Push to staging registry.
+PREFIX ?= staging-k8s.gcr.io
```

Comment:
is the space allowed here? makefiles are finnicky about spacing, and I don't recall the rules off the top of my head.

Reply:
Good catch!